src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/ocpp2/ChangeConfigurationRequest.java

```diff
@@ -0,0 +1,10 @@
+package fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp2;
+
+import fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp2.data.SetVariableDataType;
+import java.util.List;
+
+/**
+ * A message sent to the machine to change the configuration in OCPP 2.0.
+ */
+public record ChangeConfigurationRequest(List<SetVariableDataType> list) {
```

Comment:
missing doc for List<SetVariableDataType> list
where is checkstyle when we need it ?

Reply:
Checkstyle is dead
Will add

Reply:
This class is totally incorrect, I deleted it
This should be in another task !